src/transformers/image_utils.py

```diff
@@ -253,17 +276,31 @@ def center_crop(self, image, size):
         size given, it will be padded (so the returned result has the size asked).
 
         Args:
-            image (`PIL.Image.Image` or `np.ndarray` or `torch.Tensor`):
+            image (`PIL.Image.Image` or `np.ndarray` or `torch.Tensor` of shape (n_channels, height, width) or (height, width, n_channels)):
                 The image to resize.
             size (`int` or `Tuple[int, int]`):
                 The size to which crop the image.
+
+        Returns:
+            new_image: A center cropped `PIL.Image.Image` or `np.ndarray` or `torch.Tensor` of shape: (n_channels,
+            height, width).
         """
         self._ensure_format_supported(image)
+
         if not isinstance(size, tuple):
             size = (size, size)
 
         # PIL Image.size is (width, height) but NumPy array and torch Tensors have (height, width)
-        image_shape = (image.size[1], image.size[0]) if isinstance(image, PIL.Image.Image) else image.shape[-2:]
+        if is_torch_tensor(image) or isinstance(image, np.ndarray):
+            if image.ndim == 2:
+                image = self.expand_dims(image)
+            if image.shape[0] in [1, 3]:
+                image_shape = image.shape[1:]
+            else:
+                image_shape = image.shape[:2]
```

Comment:
Can be done in one line here since it's so short:
```suggestion
            image_shape = image.shape[1:] if image.shape[0] in [1, 3] else image.shape[:2]
```

Reply:
Thanks! Fixed it